Tips: not work for okhttp_3.x version

System.err: 
Caused by: java.lang.ClassCastException: okhttp3.Headers cannot be cast to java.lang.Iterable

Because of:
okhttp3.Headers will extends Iterable until after okttp_4.x.

Maybe need add to README.

--- a/README.md
+++ b/README.md
@@ -2,6 +2,9 @@
 
 Simply way to transform OkHttp requests into curl logs.
 
+## Supported OkHttp versions
+Currently Ok2Curl requires OkHttp in version 4.x.
+
 ## Usage
 Add library to project dependencies. Library is hosted on Maven Central.
 ```groovy

